utils/switch_task_to_static/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+services:
+  generate_chunks_for_task:
+    image: cvat/server:v2.45.0
+    environment:
+      CVAT_POSTGRES_HOST: cvat_db
+    volumes:
+      - cvat_data:/home/django/data
+      - ./switch_task_to_static_cache.py:/home/django/switch_task_to_static_cache.py:ro
+    networks:
+      - cvat
+    entrypoint: ["python", "switch_task_to_static_cache.py"]
+
+volumes:
+  cvat_data:
+    name: cvat_cvat_data
+    external: True
+
+networks:
+  cvat:
+    name: cvat_cvat
+    external: true
```

Comment:
Can be removed if https://github.com/cvat-ai/cvat/pull/9552/files#r2394262978 accepted